qiskit/circuit/qpy_serialization.py

```diff
@@ -275,7 +274,7 @@ class if it's defined in Qiskit. Otherwise it falls back to the custom
 
     struct {
         char type;
-        unisgned int index;
+        unsigned int index;
```

Comment:
I wasn't sure if this should've been changed to a unit with the number of bits (i.e. ``uint32_t``)?

Reply:
Oh yeah, that's a good idea I thought I updated all the ints to use a type with a consistent size across platforms but must have missed this. If you don't mind updating it that would be more descriptive than `unsigned int`